### PR TITLE
Add Android SIG

### PR DESCRIPTION
**Pull request by breedx-splk**
We are peeling off a dedicated Android SIG meeting and would like some representation on the main community page. Thanks in advance.

### CODE REVIEW

**Comment by mtwo** [comment]
Does the OTel GC account already own the notes doc? If not, can you transition ownership to cncf-opentelemetry-governance@lists.cncf.io ?

**Comment by breedx-splk** [comment]
> Does the OTel GC account already own the notes doc? If not, can you transition ownership to [cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io) ?

Nope, although I tried....

image

Maybe the move is to just make a copy form the GC account and change this PR to use that one. Can you help with that?

**Comment by trask** [comment]
> Maybe the move is to just make a copy from the GC account and change this PR to use that one

here you go: https://docs.google.com/document/d/1W72oElAJuYx3efM9wkH5l1iksdfwmTSkVl-H7KHTLnY/edit?usp=sharing

File: README.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ SIG-specific GitHub discussions.
...
 | Maintainers Meeting                   | Every Monday at 09:00PT                                                                                    | [Google Doc](https://docs.google.com/document/d/1_X2CTiDXmxk2JozBNwQ2uP8FfV1rxx9ISE5puHu0yIk/edit)                             | [#otel-maintainers](https://cloud-native.slack.com/archives/C01NJ7V1KRC)                                                                                     |
+| Android: SDK + Automatic Instrumentation | Every Tuesday at 08:00PT                                                                                | [Google Doc](https://docs.google.com/document/d/1dawuB_CO8icBuTSQp1DNhsnm1kJGolrkacg-EpkgAZ4/edit)                             | [#otel-android](https://cloud-native.slack.com/archives/C05J0T9K27Q)                                                                                    |
```
Comment (mtwo) [inline README.md:147]:
```suggestion
| Android: SDK + Automatic Instrumentation | Every Tuesday at 08:00PT                                                                                | [Google Doc][https://docs.google.com/document/d/1W72oElAJuYx3efM9wkH5l1iksdfwmTSkVl-H7KHTLnY](https://docs.google.com/document/d/1W72oElAJuYx3efM9wkH5l1iksdfwmTSkVl-H7KHTLnY/)                             | [#otel-android](https://cloud-native.slack.com/archives/C05J0T9K27Q)                                                                                    |
```

**Review by mtwo — APPROVED** [review]
@trask beat me to it, but here's a suggestion that includes his link!

**Comment by breedx-splk** [comment]
Thanks y'all. Appreciate it. 🙏🏻

**Review by trask — APPROVED** [review]

**Pull request merged**